src/framework/preferences/js/Panels.js

```diff
@@ -317,13 +296,28 @@ var fluid_2_0 = fluid_2_0 || {};
                     target: "{that > " + componentName + "}.options"
                 });
             }
-        });
 
+            // Construct the model relay btw the composite panel and its subpanels
+            var componentRelayRules = {};
+            var preferenceMap = fluid.get(fluid.prefs.compositePanel.prefetchComponentOptions(componentOptions.type, componentOptions.options), ["preferenceMap"]);
+            fluid.each(preferenceMap, function (prefObj, prefKey) {
+                $.each(prefObj, function (prefRule) {
+                    if (prefRule.indexOf("model.") === 0) {
+                        fluid.set(componentRelayRules, prefRule.slice(6), "{compositePanel}.model." + fluid.prefs.subPanel.safePrefKey(prefKey));
+                    }
+                });
+            });
+            fluid.set(relayOption, [componentName], componentRelayRules);
```

Comment:
Given the path seems to have just one segment, couldn't this more simply just be 
relayOption[componentName] = componentRelayRules?